src/util.cpp

```diff
@@ -416,6 +416,7 @@ static std::string ToDasher(const std::string &input) {
 
 std::string ConvertCase(const std::string &input, Case output_case,
                         Case input_case) {
+  if (output_case == Case::kKeep) return input;
```

Comment:
can we add a test for this, I have other ones already setup for this, so it should be easy to add.

Reply:
Done